OpenOversight/app/utils.py

```diff
@@ -114,17 +114,32 @@ def compute_hash(data_to_hash):
     return hashlib.sha256(data_to_hash).hexdigest()
 
 
+# Map file extension to content type naively
+def guess_mime_type(filename):
```

Comment:
As long as we won't be testing images, we can use a function in the standard library called [`imghdr.what`](https://docs.python.org/3/library/imghdr.html#imghdr.what). It returns a value based on the _contents_ of the file, rather than its extension.

Although I would advise against it, you can still guess based on file extension using https://docs.python.org/3/library/mimetypes.html#mimetypes.guess_extension instead.